app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php

```diff
@@ -131,6 +132,32 @@ public function form(Form $form): Form
                             ->label('Active')
                             ->hidden(fn (Get $get): bool => blank($get('logo'))),
                     ]),
+
+                Section::make('Changelog URL')
+                    ->aside()
+                    ->schema([
+                        TextInput::make('changelog_url')
+                            ->label('Changelog URL')
+                            ->url()
+                            ->maxLength('255')
+                            ->model(
+                                ThemeSettings::getSettingsPropertyModel('theme.changelog_url'),
+                            ),
+                    ])
+                    ->visible(fn () => FeatureFlag::AddUrlToThemeSettings->active()),
+
+                Section::make('Product Knowledge Base URL')
+                    ->aside()
+                    ->schema([
+                        TextInput::make('product_knowledge_base_url')
+                            ->label('Product Knowledge Base URL')
+                            ->url()
+                            ->maxLength('255')
+                            ->model(
+                                ThemeSettings::getSettingsPropertyModel('theme.product_knowledge_base_url'),
+                            ),
```

Comment:
This is only needed for things like the Spatie media input. The other inputs like this one do not need it.